packages/alore-auth-ui/src/machine/index.ts

```
@@ -1137,24 +1293,28 @@
   resolvedState = authMachine.resolveState(previousState);
 }
 
-export const authService = (services: {}) =>
-  interpret(
+export const authService = (services: {}, context: AuthMachineContext) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)